.ci/checker-framework.groovy

```diff
@@ -398,6 +396,21 @@ class CheckerFrameworkError implements Comparable<CheckerFrameworkError> {
     List<String> details
     String lineContent
     int lineNumber
+
+    /**
+     * Whether the error is unstable. Unstable errors in suppression list are not flagged as
+     * unnecessary suppressions. An error is considered to be unstable when:
+     * <ul>
+     *     <li>
+     *         Error message changes with each run. Some error messages contains strings like
+     *         {@code temp-var-1234}, the numerical part changes with each run so the error is
+     *         considered unstable. In such cases numerical values in {@code details} and
+     *         {@code message} are replaced with empty string while comparing and hashing errors.
+     *     </li>
+     *     <li>
+     *         The error is in general unstable, it sometimes appears and sometimes it does not.
```

Comment:
Is this an actual case that we deal with? I thought unstable for checker means that some variable name changes only.

Reply:
No, we do not have a case like this, removed it, otherwise would have unnecessarily confused the developer.